src/Settings.php

```diff
@@ -4,229 +4,233 @@
+			$result = [];
 
-		foreach ( $tribe_options as $key => $value ) {
-			if ( 0 === strpos( $key, $this->get_options_prefix() ) ) {
-				$result[$key] = $value;
+			foreach ( $tribe_options as $key => $value ) {
+				if ( 0 === strpos( $key, $this->get_options_prefix() ) ) {
+					$result[ $key ] = $value;
+				}
 			}
+
+			return $result;
 		}
 
-		return $result;
-	}
+		/**
+		 * Given an option key, delete this extension's option value.
+		 *
+		 * This automatically prepends this extension's option prefix so you can just do `$this->delete_option( 'a_setting' )`.
+		 *
+		 * @param string $key
+		 *
+		 * @return mixed
+		 */
+		public function delete_option( $key = '' ) {
+			$key = $this->sanitize_option_key( $key );
 
-	/**
-	 * Given an option key, delete this extension's option value.
-	 *
-	 * This automatically prepends this extension's option prefix so you can just do `$this->delete_option( 'a_setting' )`.
-	 *
-	 * @param string $key
-	 *
-	 * @return mixed
-	 */
-	public function delete_option( $key = '' ) {
-		$key = $this->sanitize_option_key( $key );
+			$options = Tribe__Settings_Manager::get_options();
 
-		$options = Tribe__Settings_Manager::get_options();
+			unset( $options[ $key ] );
 
-		unset( $options[$key] );
+			return Tribe__Settings_Manager::set_options( $options );
+		}
 
-		return Tribe__Settings_Manager::set_options( $options );
-	}
+		/**
+		 * Here is an example of removing settings from Events > Settings > General tab > "Map Settings" section
+		 * that are specific to Google Maps.
+		 */
+		public function remove_settings() {
+			// "Enable Google Maps" checkbox
+			$this->settings_helper->remove_field( 'embedGoogleMaps', 'general' );
+			// "Map view search distance limit" (default of 25)
+			$this->settings_helper->remove_field( 'geoloc_default_geofence', 'general' );
+			// "Google Maps default zoom level" (0-21, default of 10)
+			$this->settings_helper->remove_field( 'embedGoogleMapsZoom', 'general' );
+		}
 
-	/**
-	 * Here is an example of removing settings from Events > Settings > General tab > "Map Settings" section
-	 * that are specific to Google Maps.
-	 */
-	public function remove_settings() {
-		// "Enable Google Maps" checkbox
-		$this->settings_helper->remove_field( 'embedGoogleMaps', 'general' );
-		// "Map view search distance limit" (default of 25)
-		$this->settings_helper->remove_field( 'geoloc_default_geofence', 'general' );
-		// "Google Maps default zoom level" (0-21, default of 10)
-		$this->settings_helper->remove_field( 'embedGoogleMapsZoom', 'general' );
-	}
+		/**
+		 * Adds a new section of fields to Events > Settings > General tab, appearing after the "Map Settings" section and
+		 * before the "Miscellaneous Settings" section.
+		 */
+		public function add_settings() {
+			$fields = [
+				// TODO: Settings heading start. Remove this element if not needed. Also remove the corresponding `get_example_intro_text()` method below.
+				$this->opts_prefix . 'Example'   => [
```

Comment:
I would make a method to append the prefix, makes it error proof.

Reply:
done